airbyte-integrations/connectors/source-mongodb-v2/src/main/java/io/airbyte/integrations/source/mongodb/cdc/MongoDbCdcInitializer.java

```diff
@@ -165,7 +166,25 @@ public List<AutoCloseableIterator<AirbyteMessage>> createCdcIterators(
     final AutoCloseableIterator<AirbyteMessage> initialSnapshotIterator = AutoCloseableIterators.appendOnClose(
         AutoCloseableIterators.concatWithEagerClose(initialSnapshotIterators), mongoClient::close);
 
-    return List.of(initialSnapshotIterator, AutoCloseableIterators.lazyIterator(incrementalIteratorSupplier, null));
+    final List<AutoCloseableIterator<AirbyteMessage>> cdcStreamsStartStatusEmitters = incrementalOnlyStreamsCatalog.getStreams().stream()
+        .filter(stream -> !initialSnapshotStreams.contains(stream))
+        .map(stream -> {
+          LOGGER.info("*** adding STARTED stream for {}/{}", stream.getStream().getNamespace(), stream.getStream().getName());
```

Comment:
optional: feels like we don't need this log

Reply:
🙏